src/mobu/storage/process.py

```diff
@@ -0,0 +1,57 @@
+"""Class with method to execute noninteractive subprocesses asynchronously."""
+
+import asyncio
+from pathlib import Path
+from shlex import join
+
+from structlog.stdlib import BoundLogger
+
+from ..exceptions import AsyncioProcessError
+from ..models.user import AuthenticatedUser
+
+
+class Process:
+    """A thin wrapper around asyncio.subprocess.create_process_exec."""
```

Comment:
I don't think this is sufficiently generic to warrant being a separate class as opposed to a method on `Git`. It doesn't have any state that isn't already in the `Git` object state.